sdk/cosmos/azure-cosmos-spark_3-0_2-12/pom.xml

```diff
@@ -0,0 +1,340 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-spark_3-0_2-12</artifactId>
+  <version>4.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos-spark_3-0_2-12;current} -->
+  <packaging>jar</packaging>
+  <name>OLTP Spark Connector for Azure Cosmos DB SQL API</name>
+  <description>OLTP Spark Connector for Azure Cosmos DB SQL API</description>
+  <url>https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cosmos/azure-cosmos-spark_3_2.12</url>
+
+  <properties>
+    <maven.build.timestamp.format>MM-dd-HH-mm-ss</maven.build.timestamp.format>
+    <jacoco.min.branchcoverage>0.17</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0.18</jacoco.min.linecoverage>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
```

Comment:
This shouldn't be here - we don't want to get into the habit of having jacoco skips everywhere.

Reply:
Fixed in next iteration